etk/extractors/dbpedia_spotlight_extractor.py

```diff
@@ -1,25 +1,53 @@
-from etk.extractor import Extractor
+from etk.extractor import Extractor, InputType
 from etk.extraction import Extraction
 from typing import List
-
+import requests
 
 
 class DBpediaSpotlightExtractor(Extractor):
-    def __init__(self):
-        # TODO initialise stuff
-        # TODO if using the dbpedia webservice, a class variable should be set with the url
-        # TODO if using the codebase, the library can be setup to called here
-        # TODO if using the codebase, we need a local dbpedia against which the queries are to be made, set it up here as well
-        pass
 
-    def extract(self, text: str) -> List[Extraction]:
+    def __init__(self, extractor_name: str, search_url: str):
+        Extractor.__init__(self, input_type=InputType.TEXT,
+                           category="built_in_extractor",
+                           name=extractor_name)
+        self.search_url = search_url
+
+    def extract(self, text: str, confidence=0.5, filter=['Person', 'Place', 'Organisation']) -> List[Extraction]:
         """
             Extract with the input text
             Args:
                 text: str
-
+                confidence: float
```

Comment:
there is one more change, the docstring should explain each variable, and not only list its type, please update this